openassessment/xblock/test/data/self_assessment_scenario.xml

```diff
@@ -43,6 +43,7 @@
         </criterion>
     </rubric>
     <assessments>
+        <assessment name="staff-assessment" required="False" />
```

Comment:
I'm a bit worried that I had to add this to make the staff assessment available in test runs, is there some other way to specify it as always available? I'm going to check into that a bit before I write up more thorough staff tests.